lib/bolt/module_installer.rb

```diff
@@ -94,23 +94,26 @@ def add(name, modules, puppetfile_path, moduledir, config_path)
 
     # Installs a project's module dependencies.
     #
-    def install(modules, path, moduledir, force: false)
+    def install(modules, path, moduledir, force: false, resolve: true)
       require 'bolt/puppetfile'
 
       puppetfile = Bolt::Puppetfile.new(modules)
 
       # If the Puppetfile exists, check if it includes specs for each declared
       # module, erroring if there are any missing. Otherwise, resolve the
       # module dependencies and write a new Puppetfile. Users can forcibly
-      # overwrite an existing Puppetfile with the '--force' option.
-      if path.exist? && !force
-        assert_managed_puppetfile(puppetfile, path)
-      else
-        @outputter.print_message "Resolving module dependencies, this may take a moment"
-        puppetfile.resolve
-
-        @outputter.print_message "Writing Puppetfile at #{path}"
-        puppetfile.write(path)
+      # overwrite an existing Puppetfile with the '--force' option, or opt to
+      # install the Puppetfile as-is with --no-resolve
```

Comment:
```suggestion
      # Overwrite an existing Puppetfile with the '--force' option, or opt to
      # install the Puppetfile as-is with --no-resolve.
```

Reply:
This isn't the start of a sentence...

Reply:
D'oh. Sorry, I think my eyes skipped over the previous lines. :|